### PR TITLE
Remove usage is solcitor application

### PR DESCRIPTION
**Pull request by mapingo**
### Change description ###
Remove usage of isSolcitorApplication method
Refactor respondent AoS invitation generation
Remove digitalNoticeOfProceedings from AcknowledgementOfService

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/SendAosPackToRespondent.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    private AosPackPrinter aosPackPrinter;
+
+    @Override
+    public CaseDetails<CaseData, State> apply(final CaseDetails<CaseData, State> caseDetails) {
+
+        final Long caseId = caseDetails.getId();
+        final CaseData caseData = caseDetails.getData();
+
+        if (!caseData.getApplication().isSolicitorServiceMethod()) {
+            final Applicant respondent = caseData.getApplicant2();
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/SendAosPackToRespondent.java:27]: I guess this variable is redundant now, you may as well pass it straight through on line 33

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java [hunk 2]
```diff
@@ -183,6 +183,15 @@ public boolean isBasedOverseas() {
             && !("United Kingdom").equalsIgnoreCase(homeAddress.getCountry());
     }
 
+    @JsonIgnore
+    public String getCorrespondenceEmail() {
+        if (isRepresented()) {
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java:188]: My personal preference is for ternary if statements, but that is just a personal preference

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/RetiredFields.java [hunk 3]
```diff
@@ -356,6 +357,12 @@
     )
     private List<String> transformationAndOcrWarnings = new ArrayList<>();
 
+    @CCD(
+        label = "Retired Digital Notice of Proceedings?",
+        access = {CaseworkerAndSuperUserAccess.class}
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/divorce/divorcecase/model/RetiredFields.java:362]: you can remove the access here now

**Review by linusnorton — CHANGES_REQUESTED** [review]
I might have missed it but I don't think you removed isSolicitorApplication from Application.java

**Review by linusnorton — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/RetiredFields.java (hunk 3 above)
Comment (mapingo) [reply src/main/java/uk/gov/hmcts/divorce/divorcecase/model/RetiredFields.java]: Removed.
https://github.com/hmcts/nfdiv-case-api/pull/1433/commits/b22c4d5c85c97bfc2a7cb862b06279baaadfd4e3

**Pull request merged**